### ISSUE TITLE
Permit generation skipped because the issue was not closed as completed

### ISSUE THREAD

**Issue by pavlovcik** [ubiquity/ubiquibot#503]
Let's reword this to 

Permit generation skipped because this is marked as unplanned. 

> Permit generation skipped because the issue was not closed as completed

_Originally posted by @ubiquibot in https://github.com/ubiquity/pay.ubq.fi/issues/48#issuecomment-1633646803_

**Comment by pavlovcik** [comment]
> #### Task Creator Reward
> ### [ **pavlovcik: [ CLAIM 16 WXDAI ]**](https://pay.ubq.fi?claim=eyJwZXJtaXQiOnsicGVybWl0dGVkIjp7InRva2VuIjoiMHhlOTFEMTUzRTBiNDE1MThBMkNlOERkM0Q3OTQ0RmE4NjM0NjNhOTdkIiwiYW1vdW50IjoiMTYwMDAwMDAwMDAwMDAwMDAwMDAifSwibm9uY2UiOiI5NTA4MTIyMDYzMzgyMjIzMzM3NzE4NzEzMjE0NzQzOTY2NzgxODY5OTMyMzg5NjIyOTg1NjcyNTc0OTU4OTI4NTE1MzM0MjI2MDkwMyIsImRlYWRsaW5lIjoiMTE1NzkyMDg5MjM3MzE2MTk1NDIzNTcwOTg1MDA4Njg3OTA3ODUzMjY5OTg0NjY1NjQwNTY0MDM5NDU3NTg0MDA3OTEzMTI5NjM5OTM1In0sInRyYW5zZmVyRGV0YWlscyI6eyJ0byI6IjB4NDAwN0NFMjA4M2M3RjNFMTgwOTdhZUIzQTM5YmI4ZUMxNDlhMzQxZCIsInJlcXVlc3RlZEFtb3VudCI6IjE2MDAwMDAwMDAwMDAwMDAwMDAwIn0sIm93bmVyIjoiMHhmODdjYTQ1ODNDNzkyMjEyZTUyNzIwZDEyN0U3RTBBMzhCODE4YUQxIiwic2lnbmF0dXJlIjoiMHg4N2Y3YzFkYTk4OTI2NGMwMDUxNWZjNjIzY2YxZGFiNDNmNTRhNzcyNjc2Y2U4NWJhODVhNzFiOTllM2M3NGNjNjk5YzI4Njc5NWQ2Mzc2MTg4MmRiMzcyNTExM2RiMWYyMDRiYTE5ZTNhMWZlOTJhMjNkMjJjNzM1NTM2ZTFjNDFjIn0=&network=100)

This seems too high. I believe it is crediting the italicized text but I just removed credit for italics. 

https://github.com/ubiquity/ubiquibot-config/blob/development/.github/ubiquibot-config.yml

@whilefoo rfc

**Comment by whilefoo** [comment]
If I understand correctly the text inside italics (and anchor) should not be credited only when config is `i: 0` or even if italics have reward?

**Comment by whilefoo** [comment]
Ok so this behavior should be for all elements, if an element has reward `0` then it should not count the text inside. Because currently even if all elements have `0` reward and word reward is `1` it will credit every word.

**Comment by pavlovcik** [comment]
> Ok so this behavior should be for all elements, if an element has reward `0` then it should not count the text inside. Because currently even if all elements have `0` reward and word reward is `1` it will credit every word.

Interesting problem. I imagine that we need some type of special value to instruct to the parser to exclude it from the calculation entirely. Perhaps we can use `null` or even `ignore`?

For example:
```
ul: 0
li: 0.5
i: ignore
```
```
- this is a _test_
```
Would yield `0.5 + ( 0.1 * 3 ) = 0.8`, ignoring "_test_"

**Comment by whilefoo** [comment]
That might be a little bit awkward for partners because default config will be `ignore` so if they want to count the words inside paragraphs they have to explicitly set `p: 0`. Unless we make some sensible defaults.

**Comment by pavlovcik** [comment]
- https://github.com/ubiquity/ubiquibot/issues/772
